tests/test_transcribe.py

```diff
@@ -10,7 +10,7 @@
 @pytest.mark.parametrize("model_name", whisper.available_models())
 def test_transcribe(model_name: str):
     device = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
we should add a simlar test for hpu as well

Reply:
Thanks, just added a new test for `hpu` - https://github.com/BlueLabelLabs/whisper_fork/pull/2/files#diff-ccf3e38e2062ec2b77cbeda0fb709b927859e107a1d84463651fd53915182616R47